README.md

```diff
@@ -35,29 +35,42 @@ pip install neural-compressor
 > More installation methods can be found at [Installation Guide](https://github.com/intel/neural-compressor/blob/master/docs/source/installation_guide.md). Please check out our [FAQ](https://github.com/intel/neural-compressor/blob/master/docs/source/faq.md) for more details.
 
 ## Getting Started
-### Quantization with Python API
 
-```shell
-# Install Intel Neural Compressor and TensorFlow
-pip install neural-compressor
-pip install tensorflow
-# Prepare fp32 model
-wget https://storage.googleapis.com/intel-optimized-tensorflow/models/v1_6/mobilenet_v1_1.0_224_frozen.pb
+### Weight-Only Quantization
+```python
+# Get the float model
+from transformers import AutoModel
+
+float_model = AutoModel.from_pretrained("bert-base-uncased")
+
+# Apply weight-only quantization to the model
+from neural_compressor.quantization import fit
+from neural_compressor.config import PostTrainingQuantConfig
+
+woq_conf = PostTrainingQuantConfig(approach="weight_only")
+quantized_model = fit(model=float_model, conf=woq_conf)
 ```
+
+### Static Quantization
```

Comment:
Static Quantization (Non-LLMs)

Reply:
Thanks, updated in [36e8c81](https://github.com/intel/neural-compressor/pull/1670/commits/36e8c81eafb785997b13b9edb7c85c119d490a39).